utils/build-windows.bat

```diff
@@ -345,4 +348,73 @@ goto :eof
 endlocal
 
 
+:download_xml2
+:: Downloads libxml2 from the Azure artifacts
+setlocal enableextensions enabledelayedexpansion
+
+set pipeline_id=10
+set build_name=xml2-windows-x64
+set file_name=%build_name%.zip
+
+for /f "tokens=*" %%G IN ('curl "https://dev.azure.com/compnerd/windows-swift/_apis/build/builds?definitions=%pipeline_id%&resultFilter=succeeded,partiallySucceeded&$top=1&api-version-string=5.0" ^| jq ".value[0].id"') DO (
+    for /f "tokens=*" %%H IN ('curl "https://dev.azure.com/compnerd/windows-swift/_apis/build/builds/%%G/artifacts?api-version-string=5.0" ^| jq -r ".value[] | select(.name == \"%build_name%\") | .resource.downloadUrl"') DO (
+        curl -L -o %file_name% "%%H"
+    )
+)

```

Comment:
This is getting pretty involved.  Is there a reason to not just prefer `windows-swift.py` from compnerd/windows-swift?  That will allow you to download the latest artifacts.

Reply:
Is not really that involved, it is that Batch files are crazy. What could be solved in three simple lines in Bash, needs this `for /f` crazyness in Batch.

I just wanted to see if it can be done, and not depend on anything. The ideal situation would be using the most official versions of this dependencies. I have tracked down some libxml2 binaries that are semi-official, and include zlib and iconv (something that Azure is missing). The problem is then libcurl which seems to be statically linked against zlib.

Reply:
I think that there may be issues with iconv on Windows.  Where did you find the semi-official libxml2 builds?  Yes, libcurl is statically linked to zlib to avoid the need for additional libraries at runtime.